Error while building project with Umbraco.AI.Agent.Copilot

### Which component is this issue related to?

Umbraco.AI.Agent.Copilot

### Which Umbraco AI version are you using? (Please write the *exact* version, example: 10.1.0)

1.0.0-alpha5

### Bug summary

While building the project with a reference to the package Umbraco.AI.Agent.Copilot, I get build errors concerning the tsconfig.json in the package.

I could solve the problem by deleting the tsconfig.json from the project in visual studio (/Client/tsconfig.json).

### Specifics

```
tsc : error TS18003: Build:No inputs were found in config file 'C:/Users/mbrunner/.nuget/packages/umbraco.ai.agent.copilot/1.0.0-alpha5/contentFiles/any/net10.0/Client/tsconfig.json'. Specified 'include' paths were '["src"]' and 'exclude' paths were '[]'.
.nuget\packages\umbraco.ai.agent.copilot\1.0.0-alpha5\contentFiles\any\net10.0\Client\tsconfig.json(11,29): error TS6046: Build:Argument for '--moduleResolution' option must be: 'node', 'classic'.
.nuget\packages\umbraco.ai.agent.copilot\1.0.0-alpha5\contentFiles\any\net10.0\Client\tsconfig.json(12,9): error TS5023: Build:Unknown compiler option 'allowImportingTsExtensions'.
.nuget\packages\umbraco.ai.agent.copilot\1.0.0-alpha5\contentFiles\any\net10.0\Client\tsconfig.json(14,9): error TS5023: Build:Unknown compiler option 'moduleDetection'.
``` 

### Steps to reproduce

```powershell
dotnet new umbraco --name UmbracoAiTest
dotnet add package Umbraco.AI --version 1.6.0
dotnet add package Umbraco.AI.Agent --version 1.5.0
dotnet add package Umbraco.AI.Agent.Copilot --version 1.0.0-alpha5
dotnet add package Umbraco.AI.MicrosoftFoundry --version 1.2.0
dotnet add package Umbraco.AI.Prompt --version 1.5.0
``` 
Try to compile the Project.

### Expected result / actual result

_No response_

### Dependencies

_No response_

--- a/Umbraco.AI.Agent.Copilot/src/Umbraco.AI.Agent.Copilot/Umbraco.AI.Agent.Copilot.csproj
+++ b/Umbraco.AI.Agent.Copilot/src/Umbraco.AI.Agent.Copilot/Umbraco.AI.Agent.Copilot.csproj
@@ -22,8 +22,8 @@
     <ItemGroup>
         <!-- Dont include the client folder as part of packaging nuget build -->
         <Content Remove="Client\**" />
-        <Content Include="Client\package.json" />
-        <Content Include="Client\tsconfig.json" />
+        <Content Include="Client\package.json" Pack="false" />
+        <Content Include="Client\tsconfig.json" Pack="false" />
 
         <!-- However make the Umbraco-package.json included for dotnet pack or nuget package and visible to the solution -->
         <None Include="Client\public\umbraco-package.json" Pack="false" />

--- a/Umbraco.AI.Agent.UI/src/Umbraco.AI.Agent.UI/Umbraco.AI.Agent.UI.csproj
+++ b/Umbraco.AI.Agent.UI/src/Umbraco.AI.Agent.UI/Umbraco.AI.Agent.UI.csproj
@@ -22,8 +22,8 @@
     <ItemGroup>
         <!-- Dont include the client folder as part of packaging nuget build -->
         <Content Remove="Client\**" />
-        <Content Include="Client\package.json" />
-        <Content Include="Client\tsconfig.json" />
+        <Content Include="Client\package.json" Pack="false" />
+        <Content Include="Client\tsconfig.json" Pack="false" />
 
         <!-- However make the Umbraco-package.json included for dotnet pack or nuget package and visible to the solution -->
         <None Include="Client\public\umbraco-package.json" Pack="false" />

--- a/Umbraco.AI.Agent/src/Umbraco.AI.Agent.Web.StaticAssets/Umbraco.AI.Agent.Web.StaticAssets.csproj
+++ b/Umbraco.AI.Agent/src/Umbraco.AI.Agent.Web.StaticAssets/Umbraco.AI.Agent.Web.StaticAssets.csproj
@@ -9,8 +9,8 @@
     <ItemGroup>
         <!-- Dont include the client folder as part of packaging nuget build -->
         <Content Remove="Client\**" />
-        <Content Include="Client\package.json" />
-        <Content Include="Client\tsconfig.json" />
+        <Content Include="Client\package.json" Pack="false" />
+        <Content Include="Client\tsconfig.json" Pack="false" />
 
         <!-- However make the Umbraco-package.json included for dotnet pack or nuget package and visible to the solution -->
         <None Include="Client\public\umbraco-package.json" Pack="false" />

--- a/Umbraco.AI.Prompt/src/Umbraco.AI.Prompt.Web.StaticAssets/Umbraco.AI.Prompt.Web.StaticAssets.csproj
+++ b/Umbraco.AI.Prompt/src/Umbraco.AI.Prompt.Web.StaticAssets/Umbraco.AI.Prompt.Web.StaticAssets.csproj
@@ -9,8 +9,8 @@
     <ItemGroup>
         <!-- Dont include the client folder as part of packaging nuget build -->
         <Content Remove="Client\**" />
-        <Content Include="Client\package.json" />
-        <Content Include="Client\tsconfig.json" />
+        <Content Include="Client\package.json" Pack="false" />
+        <Content Include="Client\tsconfig.json" Pack="false" />
 
         <!-- However make the Umbraco-package.json included for dotnet pack or nuget package and visible to the solution -->
         <None Include="Client\public\umbraco-package.json" Pack="false" />

--- a/Umbraco.AI/src/Umbraco.AI.Web.StaticAssets/Umbraco.AI.Web.StaticAssets.csproj
+++ b/Umbraco.AI/src/Umbraco.AI.Web.StaticAssets/Umbraco.AI.Web.StaticAssets.csproj
@@ -10,11 +10,11 @@
     <ItemGroup>
         <!-- Dont include the client folder as part of packaging nuget build -->
         <Content Remove="Client\**" />
-        <Content Include="Client\api-extractor.json" />
-        <Content Include="Client\package.json" />
-        <Content Include="Client\tsconfig.api.json" />
-        <Content Include="Client\tsconfig.app.json" />
-        <Content Include="Client\tsconfig.json" />
+        <Content Include="Client\api-extractor.json" Pack="false" />
+        <Content Include="Client\package.json" Pack="false" />
+        <Content Include="Client\tsconfig.api.json" Pack="false" />
+        <Content Include="Client\tsconfig.app.json" Pack="false" />
+        <Content Include="Client\tsconfig.json" Pack="false" />
 
         <!-- However make the Umbraco-package.json included for dotnet pack or nuget package and visible to the solution -->
         <None Include="Client\public\umbraco-package.json" Pack="false" />

